### PR TITLE
Project 2: Components.

### PR DESCRIPTION
**Pull request by muhammedb8826**
# I have created and implemented the following
- Added the components directory inside the src directory
- Created Calculator.js component
- Imported Calculator.js inside App.js
- Added styles to match the design
- Run the project to the browser successfully

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,59 @@
-.App {
```
Comment (tresorsawasawa) [inline src/App.css:1]: - [ ] As required for the project, kindly make sure all the boilerplate that got created by the CRA (texts, images, and styles) should be deleted. Right now, I can that you still have files non related files created by CRA such as `src/reportWebVitals.js`, `src/setupTests.js`, `App.test.js`,... I suggest you delete them

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (tresorsawasawa) [inline src/App.js:9]: - [ ] `README.md` file: Kindly make sure you update your some of your README sections with appropriate content information to make your documentation more professional. Such as:
```
## Setup
Clone this repository to your desired folder:

git clone https://github.com/muhammedb8826/Calculator.git

cd Calculator
```
```
## Live Demo
Coming soon!
Link to github repository.

------------> [To-Do list app](https://github.com/muhammedb8826/Calculator)
```
| Setup | Live Demo |
| -------------| --------------- |
| Screenshot from 2023-03-22 22-42-11 | Screenshot from 2023-03-22 22-47-47 |

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @muhammedb8826 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @muhammedb8826 

Your project is complete! Great job making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- git-flow ok
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**